src/plugins/workspace/public/components/workspace_creator/workspace_creator_form.tsx

```diff
@@ -110,29 +118,31 @@ export const WorkspaceCreatorForm = (props: WorkspaceCreatorFormProps) => {
           {/* SelectDataSourcePanel is only visible for dashboard admin and when data source is enabled*/}
           {isDashboardAdmin && isDataSourceEnabled && (
             <>
-              <EuiTitle
-                {...generateRightSidebarScrollProps(RightSidebarScrollField.DataSource)}
-                size="s"
-              >
-                <h3>
-                  {i18n.translate('workspace.creator.form.associateDataSourceTitle', {
-                    defaultMessage: 'Associate data sources',
+              <EuiPanel>
+                <EuiTitle
```

Comment:
Can we use `EuiSelect + h2` as panel title [like this](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/workspace/public/components/workspace_creator/creator_details_panel.tsx#L42-L48) to align with the guidance.

Reply:
ok